client/__tests__/e2e/annotation.e2e.test.ts

```diff
@@ -0,0 +1,696 @@
+/**
```

Comment:
These playwright specs replace the old puppeteer smoke coverage. They run each action (create, rename, delete, undo/redo) against both the main and side graph instances, so if you want to sanity-check new UI handlers this is a great place to start.